fraud/service.go

```diff
@@ -103,8 +116,12 @@ func (f *ProofService) Subscribe(proofType ProofType) (_ Subscription, err error
 	}
 	subs, err := t.Subscribe()
 	if err != nil {
+		span.RecordError(err)
+		span.SetStatus(codes.Error, "error during subscribing to a pubsub")
 		return nil, err
 	}
+	go observeFraudProofs(proofType, f.ds)
```

Comment:
This will make every subscription run observer for the same fraud type...

Also, let's make it similar to the header pkg, where it has to be:
* A public func
* Working over public API of fraud service
	* Extend API if necessary
* Optionable on the node